lib/existing-eks-opensource-observability-pattern/istio/istiocniAddon.ts

```diff
@@ -0,0 +1,26 @@
+import 'source-map-support/register';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
```

Comment:
Why do we need this file as duplicate. We can refer to class file in Single-New folder?